src/pres/loader/PGNLoader.js

```diff
@@ -208,6 +211,43 @@ export default class PGNLoader extends React.Component {
         this.props.variantChange(newVariant)
         trackEvent(Constants.EVENT_CATEGORY_PGN_LOADER, "VariantChange", newVariant)
     }
+    getInitiallyExpandedPanel(selectedSite, playerName) {
```

Comment:
I'm not sure about naming ("initial"? "initially"?) for this and getTimeControls